components/zcash_address/README.md

```diff
@@ -0,0 +1,20 @@
+# zcash_address
+
+TBD
```

Comment:
Zcash address parsing and serialization. This library allows its users to easily
recognize and give good error messages for new Zcash address types.